test/browser_tests/spec_suites/organisms/pagination.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var Blog = require(
+    '../../page_objects/page_blog.js'
+  );
+
+describe( 'Pagination', function() {
+  var page;
+
+  beforeEach( function() {
+    page = new Blog();
+    page.get();
+  } );
+
+  it( 'should navigate to the second page', function() {
+    page.paginationNextBtn.click()
+    browser.sleep( 1000 );
```

Comment:
Nope, you have to wait for the next page to load, and there doesn't seem to be anything within Protractor that does this because the content itself is mostly the same (we could never rely on the results because they change too often).